tests/EdgeToEdge/Routes/HandlePayPalPaymentNotificationRouteTest.php

```diff
@@ -352,7 +385,7 @@ public function testGivenNegativeTransactionFee_exceptionIsThrown(): void {
 			);
 
 			$this->assertSame(
-				[ 'Unhandled PayPal instant payment notification' ],
```

Comment:
Both occurrences of this (unhandled from inside the uses cases) have been converted to tests against the new logic - so there is no more test for the guard inside handleNotification, right?

Reply:
If I am understanding what you mean correctly, which I am not very sure of, then this is the test covering the guard:

![image](https://user-images.githubusercontent.com/146040/34149907-07569416-e4a7-11e7-9ee4-b277d79fd422.png)